cmd/ops_agent_windows/main_windows.go

```diff
@@ -21,10 +27,24 @@ func main() {
 	} else if err != nil {
 		log.Fatalf("failed to talk to service control manager: %v", err)
 	} else {
-		if err := install(); err != nil {
-			log.Fatal(err)
+		flag.Parse()
+		if *installServices && *uninstallServices {
+			log.Fatal("Can't use both --install and --uninstall")
+		}
+		if *installServices {
+			if err := install(); err != nil {
+				log.Fatal(err)
+			}
+			log.Printf("installed services")
+		} else if *uninstallServices {
+			if err := uninstall(); err != nil {
+				log.Fatal(err)
+			}
+			log.Printf("uninstalled services")
+		} else {
+			// TODO: add an interactive GUI box with the Install, Uninstall, and Cancel buttons.
+			fmt.Println("Invoked as a standalone program with no flags. Nothing to do.")
```

Comment:
Does it make sense to mention that either `--install` or `--uninstall` is required in the message?

Taking a step back, there was some discussion offline around this that we decided to postpone till this PR. Quoting:

```
Sounds like what we agreed are:
* The binary supports both install and uninstall.
* User specifies a flag --uninstall to uninstall.

The questionable choice is between:
* User specifies no flag to indicate they want to install (this includes starting the services)
* User has to explicit specify --install to install

And we want to match common Windows practice.
```

Reply:
I think we were all happy with having a MsgBox come up. In the meantime I'd like to preserve the existing behavior, so if you don't want to do this in the PR, please change the default for install to true.

Reply:
Is there any way to prevent introducing a breaking change when we add a MsgBox?

Reply:
To clarify it a bit, the reason I asked the question above is because we probably need a way for users to invoke the installation via PowerShell etc. without interactive confirmation so that users can script it. When we add a MsgBox, it should not break this use case.

Reply:
Oh I see. I thought the flag `--install` and `--uninstall` are directly used by users. If that's the case, it's fine either way.

Reply:
Printing a message to stdout/stderr does not work in "the GUI case" because double-clicking on the exe will not open a terminal.

Reply:
Discussed offline. Whether to support a GUI / using the exe directly is something we need to decide. If we do decide to support, the ideal UX can be controversial and we need more discussion. Let's punt on supporting those for now. Erroring out and noop when clicking on the exe is better for now, so that if we do decide to support such use cases later, we have the freedom to design the UX without worrying about introducing a breaking change.